Makefile

```diff
@@ -12,7 +13,7 @@ EMBEDDED_STATIC = generated/statik/statik.go
 default: kubeapps
 
 kubeapps: $(EMBEDDED_STATIC)
-	CGO_ENABLED=0 $(GO) build -i -o $(BINARY) $(GO_FLAGS) $(IMPORT_PATH)
+	CGO_ENABLED=$(CGO_ENABLED) $(GO) build -i -o $(BINARY) $(GO_FLAGS) $(IMPORT_PATH)
```

Comment:
is this needed? wouldn't it already be in the env?

Reply:
it does not exist in the env in the default case.

Reply:
right, but the default is CGO_ENABLED=1 right? if you ran `go build` without the env var, it shouldn't try to install anything?